flink-ai-flow/ai_flow/scheduler_service/service/workflow_event_processor.py

```diff
@@ -67,17 +68,21 @@ def run(self) -> None:
                 logging.error("Unexpected Exception", exc_info=e)
 
     def _process_event(self, event: BaseEvent):
-        # ignore scheduler event
-        if event.namespace == 'scheduler':
-            return
-        project_name = event.namespace
-        subscribed_workflow = self._get_subscribed_workflow(event, project_name)
-
-        for workflow in subscribed_workflow:
+        subscribed_workflow = []
+        projects = self.store.list_project(sys.maxsize, 0)
+        for project_name in projects:
+            workflows = self._get_subscribed_workflow(event, project_name)
+            if len(workflows) > 0:
+                for workflow in workflows:
+                    subscribed_workflow.append((project_name, workflow))
```

Comment:
We can use the `extend` method to make the code a little bit more consice.

Reply:
`extend` is different from `append` as `extend` only accepts `list` arg. 
Here `subscribed_workflow` is a list of tuple and the codes aim to add a new tuple into the list. `extend` can also work if we write codes like: `subscribed_workflow.extend([(project_name, workflow)])`. But it seems somehow not clean enough.

Reply:
I overlook the fact that it is a tuple we are adding to the list. We can keep it.